### PR TITLE
Gave some more description on software

### PR DESCRIPTION
**Pull request by thapapinak**
Gave Description or updated the description about the software on read me.

### CODE REVIEW

**Comment by blenderskool** [comment]
I don't think updating the readme with some Wikipedia content is a great idea. Also, this repo is not something that anyone would want to use, as it is just a basic GitHub Pages website for documenting my add-ons.

File: index.html [hunk 1]
```diff
@@ -51,123 +51,124 @@
...
+
+        </div>
+    </nav>
+
+</section>
+
+<section class="engine"><a rel="external" href="https://mobirise.com">Mobirise</a></section><section class="mbr-section mbr-section-hero mbr-section-full mbr-parallax-background mbr-after-navbar" id="header1-1" style="background-image: url(assets/images/background-2000x1453.jpg);">
+
+    <div class="mbr-overlay" style="opacity: 0.2; background-color: rgb(0, 0, 0);"></div>
+    <div class="mbr-overlay" style="opacity: 0.2; background-color: rgb(40, 80, 90);"></div>
```
Comment (blenderskool) [inline index.html]: Background Image without any more overlay colors looks much better. Making the background more dull.

**Comment by blenderskool** [comment]
As I mentioned, code of this repo is software generated. So if I merge your changes, later if I update the site, again the repo would come to its previous state, hence there is no point of contributing to this repo.

(Also, I will be resetting this repo, and put a completely new site which would be purely coded in HTML, CSS, and Javascript, so you may think of contributing at that time)

File: index.html (hunk 1 above)
Comment (thapapinak) [reply index.html]: I have made it with it without overlay.

**Pull request closed without merging**